symplyphysics/laws/thermodynamics/work_of_ideal_gas_in_isothermal_process.py

```diff
@@ -0,0 +1,45 @@
+from sympy import (Eq, solve, log)
+from symplyphysics import (units, Quantity, Symbol, print_expression, validate_input,
+    validate_output)
+
+## Description
+## The isothermal process of expansion (or compression) of a gas can occur under conditions where heat exchange between the gas and the external environment is carried out at a constant temperature difference.
+## To do this, the heat capacity of the external environment must be large enough, and the expansion (or compression) process must be slow enough.
+
+## Law: A = (m / mu) * R * T * ln(V_2 / V_1)
```

Comment:
There are various conventions about who makes the work. Add a note, who does the work here and what positive and negative sign means.